src/lib/Refiner.ml

```diff
@@ -518,6 +519,11 @@ struct
     let* body = tac_body fam_src in
     let* fam_trg = EM.lift_ev (Nbe.eval_tp @@ S.El (S.Ap (fam, trg))) in
     EM.ret (S.Coe (fam, src, trg, body), fam_trg)
+
+  (* todo/iev : chk_tac or syn_tac? *)
+  (* univ_tac @@ fun _ -> EM.ret @@ S.Cof (Cof.Top) *)
+  let topc : T.syn_tac = EM.ret @@ (S.Cof (Cof_tl.Top), D.Univ)
```

Comment:
I think the error will be fixed if you return `D.TpCof` in the second component here --- the meaning of `syn_tac` is to return a term together with its type, and the type of `tt` is `cof`.

Reply:
yep, that got it!